engine-rest/engine-rest-openapi/src/main/templates/paths/deployment/create/post.ftl

```diff
@@ -12,52 +12,51 @@
 
   <@lib.requestBody
       mediaType = "multipart/form-data"
-      dto = "MultiFormDeploymentDto"
-      examples = ['"example-1": {
-                     "summary": "POST `/deployment/create`",
-                     "value": [
-                       {
-                           "links": [
-                               {
-                                   "method": "GET",
-                                   "href": "http://localhost:38080/rest-test/deployment/aDeploymentId",
-                                   "rel": "self"
-                               }
-                           ],
-                           "id": "aDeploymentId",
-                           "name": "aName",
-                           "source": "process application",
-                           "deploymentTime": "2013-01-23T13:59:43.000+0200",
-                           "tenantId": null,
-                           "deployedProcessDefinitions": {
-                               "aProcDefId": {
-                                   "id": "aProcDefId",
-                                   "key": "aKey",
-                                   "category": "aCategory",
-                                   "description": "aDescription",
-                                   "name": "aName",
-                                   "version": 42,
-                                   "resource": "aResourceName",
-                                   "deploymentId": "aDeploymentId",
-                                   "diagram": "aResourceName.png",
-                                   "suspended": true,
-                                   "tenantId": null,
-                                   "versionTag": null
-                               }
-                           },
-                           "deployedCaseDefinitions": null,
-                           "deployedDecisionDefinitions": null,
-                           "deployedDecisionRequirementsDefinitions": null
-                       }
-                     ]
-                   }' ] />
+      dto = "MultiFormDeploymentDto" />
```

Comment:
I think we could include the Post Request example here. Even though it will currently not be displayed by Swagger UI other endpoints (setLocalExecutionVariableBinary, addAttachment,...) include them

Reply:
I will skip the example for now. We can revise it when working on CAM-13105.